install.sh

```diff
@@ -0,0 +1,115 @@
+set -eux;
+
+export PYTHON_VERSION=3.12.2
+
+# Install buildpack-deps packages
+apt-get install -y \
+    autoconf \
+    automake \
+    bzip2 \
+    default-libmysqlclient-dev \
+    dpkg-dev \
+    file \
+    g++ \
+    gcc \
+    imagemagick \
+    libbz2-dev \
+    libc6-dev \
+    libcurl4-openssl-dev \
+    libdb-dev \
+    libevent-dev \
+    libffi-dev \
+    libgdbm-dev \
+    libglib2.0-dev \
+    libgmp-dev \
+    libjpeg-dev \
+    libkrb5-dev \
+    liblzma-dev \
+    libmagickcore-dev \
+    libmagickwand-dev \
+    libmaxminddb-dev \
+    libncurses5-dev \
+    libncursesw5-dev \
+    libpng-dev \
+    libpq-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    libtool \
+    libwebp-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    libyaml-dev \
+    make \
+    patch \
+    unzip \
+    xz-utils \
+    zlib1g-dev \
+    tk-dev \
+    uuid-dev
+
+# Get python source code
+wget -O python.tar.xz "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz"
+
+mkdir -p /usr/src/python
+
+tar --extract --directory /usr/src/python --strip-components=1 --file python.tar.xz
+
+rm -f python.tar.xz
+
+# Compile python from source
+cd /usr/src/python
+
+./configure --build="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" --enable-loadable-sqlite-extensions --enable-optimizations --enable-option-checking=fatal --enable-shared --with-lto --with-system-expat --without-ensurepip
+
+make -j "$(nproc)" "EXTRA_CFLAGS=$(dpkg-buildflags --get CFLAGS)" "LDFLAGS=$(dpkg-buildflags --get LDFLAGS)"
+
+# https://github.com/docker-library/python/issues/784
+# prevent accidental usage of a system installed libpython of the same version
+rm -f python
+
+make -j "$(nproc)" "EXTRA_CFLAGS=$(dpkg-buildflags --get CFLAGS)" "LDFLAGS=$(dpkg-buildflags --get LDFLAGS),-rpath='\$\$ORIGIN/../lib'" python
+
+make install
+
+# enable GDB to load debugging data: https://github.com/docker-library/python/pull/701
```

Comment:
Do we need this? It's already a heavy recipe

Reply:
Don't know ... very fast to execute ... but I don't mind to remove it, I just don't really get what it is used for, and I'm always reluctant at dropping things I might regret later.

Reply:
Well I don't care either ; will all be gone anyway in the future

Reply:
yep